src/app/api/payments/pawapay/routes.ts

```
@@ -0,0 +1,279 @@
+      { error: error.message || 'Failed to process request' }, 
+      { status: 500 }
+    );
+  }
+}
+
+// This is a webhook handler for callbacks from PawaPay
+export async function handleCallback(request: NextRequest) {
+  try {
+    const body = await request.json();
+    const headers = Object.fromEntries(request.headers.entries());
+    
```

ESLint

[error] Trailing spaces not allowed.

(style/no-trailing-spaces)